app/assets/stylesheets/govuk-component/_related-items.scss

```diff
@@ -0,0 +1,24 @@
+.govuk-related-items {
```

Comment:
Does this give a standard whispace/margin to the bottom of the component? The `em` values don't seem to map to our usual `$gutter` variables, if we're porting them across, should we standardise?